nibabel/tests/test_diff.py

```diff
@@ -0,0 +1,67 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+""" Test scripts
+
+Test running scripts
+"""
+from __future__ import division, print_function, absolute_import
+
+from os.path import (dirname, join as pjoin, abspath)
+
+from hypothesis import given
+import hypothesis.strategies as st
+
+
+DATA_PATH = abspath(pjoin(dirname(__file__), 'data'))
+
+from nibabel.cmdline.diff import diff_values
+
+# TODO: MAJOR TO DO IS TO FIGURE OUT HOW TO USE HYPOTHESIS FOR LONGER LIST LENGTHS WHILE STILL CONTROLLING FOR OUTCOMES
+
+@given(st.data())
+def test_diff_values_int(data):
+    x = data.draw(st.integers(), label='x')
+    y = data.draw(st.integers(min_value = x + 1), label='x+1')
+    z = data.draw(st.integers(max_value = x - 1), label='x-1')
+
+    assert diff_values(x, x) is False
+    assert diff_values(x, y) == True
+    assert diff_values(x, z) == True
+    assert diff_values(y, z) == True
+
+
+@given(st.data())
+def test_diff_values_float(data):
+    x = data.draw(st.just(0), label='x')
+    y = data.draw(st.floats(min_value = 1e8), label='y')
+    z = data.draw(st.floats(max_value = -1e8), label='z')
+
+    assert diff_values(x, x) is False
+    assert diff_values(x, y) == True
```

Comment:
no need for "== True" since `assert` would cast the value anyways into bool.  So you could just say `assert not something_to_be_expected_to_beFalse` and `assert something_to_be_expected_to_beTrue`